flexget/plugins/input/sickbeard.py

```diff
@@ -63,11 +65,13 @@ def on_task_input(self, task, config):
         json = task.requests.get(url).json()
         entries = []
         for id, show in json['data'].items():
-            entry = Entry(title=show['show_name'],
-                          url='',
-                          series_name=show['show_name'],
-                          tvdb_id=show['tvdbid'],
-                          tvrage_id=show['tvrage_id'])
+            if show['paused'] == 0 or not config.get('only_monitored'):
```

Comment:
I assume 'paused' is either 1 or 0? In that case it'd be more pythonic to write `if not show['paused']`.